tests/spec/fs.mkdir.spec.js

```diff
@@ -88,7 +89,7 @@ describe('fs.promises.mkdir', function () {
   });
 
   it('should return a promise', function () {
-    var fs = util.fs();
+    const fs = util.fs();
     expect(fs.promises.mkdir('/tmp')).to.be.a('promise');
   });
 });
```

Comment:
One tip: it's a good idea to always include a newline at the end of all your files (I know this isn't your mistake, it was already like this).  We do this, and GitHub shows its absence, because we want to avoid having concatenated files break when the end of one file and the start of another get joined.  Just a tip for next time.

Reply:
Sounds good, will do.